resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -62,15 +62,20 @@ public DnsNameResolverBuilder(EventLoop eventLoop) {
         this.eventLoop = eventLoop;
     }
 
+    @SuppressWarnings("unchecked")
+    private T cast() {
```

Comment:
Could we rename this method to `self()`, like we call it in `AbstractHttp2ConnectionHandlerBuilder`?